src/devices/StUsb4500/Enumerations/UsbCCableConnection.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
```

Comment:
we have recently converted to 2 line copyright header (remove third line without modification to the first two)